packages/perseus-core/src/data-schema.ts

```diff
@@ -1202,9 +1202,6 @@ export type PerseusNumericInputWidgetOptions = {
     rightAlign?: boolean;
     // Always false.  Not used for this widget
     static: boolean;
-    // Used by examples, maybe not used and should be removed in the future
-    // see TODO in numeric-input
-    answerForms?: ReadonlyArray<PerseusNumericInputAnswerForm>;
```

Comment:
Since this is our contract file, are there folks we need to inform about changing it? Or this isn't actually used anywhere anymore so it shouldn't really cause an impact?

Reply:
In this case, it shouldn't have an impact. The data still can have `answerForms`, we just won't be using it. During a backfill, we'll want to remove it from the data.